mage_ai/streaming/sources/nats_js.py

```diff
@@ -21,6 +21,12 @@ class SSLConfig:
     check_hostname: bool = False
 
 
+@dataclass
+class ConsumerType:
```

Comment:
The type of this class should be enum.Enum. It doesn't need the dataclass decorator.
```
class ConsumerType(str, enum.Enum):
```

Reply:
thx, updated 🙌🏻